handlers/handlers.go

```diff
@@ -419,7 +412,8 @@ func readPreviousReleases(w http.ResponseWriter, req *http.Request, cfg *config.
 		return
 	}
 
-	validatedQueryParams, validationErrs := data.ReviewDataAggregationQueryWithParams(ctx, cfg, sanitisedParams)
+	validatedQueryParams, validationErrs := data.ReviewDataAggregationQueryWithParams(ctx, cfg, sanitisedParams, urlPath)
```

Comment:
Shouldn't this now be using func `ReviewPreviousReleasesQueryWithParams`

Reply:
yes it should! - I can't believe I missed that